### PR TITLE
Update triplea_maps.yaml

### PR DESCRIPTION
**Pull request by Cernelius**

### CODE REVIEW

**Comment by Cernelius** [comment]
I see that:
`All the default changes are options enabled as default:`
is in green.
Is it possible to use colons in the descriptions?

**Comment by RoiEXLab** [comment]
@Cernelius No worries, this is just a GitHub bug.
@ssoloff Implemented a check recently which checks if the yaml file contains valid content

File: triplea_maps.yaml [hunk 1]
```diff
@@ -1131,6 +1131,12 @@
     <br>WWIIv3 1942 - Combat-Move Before Purchasing  (by Talibush)
     <br>  1942 except where you do your combat movement before making purchases.
     <br>
+    <br>WWIIv3 1941 Move-Buy-Move (by Cernel)
+    <br>  A mod of WWIIv3 1941 where you can Combat Move both before and after purchasing units, using your normal total movement abilities in either, for easier gameplay.
+    <br>
+    <br>WWIIv3 1942 Move-Buy-Move (by Cernel)
+    <br>  A mod of WWIIv3 1942 where you can Combat Move both before and after purchasing units, using your normal total movement abilities in either, for easier gameplay.
+    <br>
```
Comment (RoiEXLab) [inline triplea_maps.yaml]: Those lines are duplicated

Comment (Cernelius) [inline triplea_maps.yaml]: Duplicated? You mean with the similar previous mod description? This is 1942 and the previous one is 1941.

Comment (RoiEXLab) [inline triplea_maps.yaml]: Oh, sorry I didn't notice the numbers were different...
Nevermind then...

**Pull request merged**